stack/http_requests/docker.keycloak.http

```diff
@@ -2,11 +2,7 @@
 POST http://localhost:8100/auth/realms/wls_realm/protocol/openid-connect/token
```

Comment:
an dem File sollte es in Bezug auf das Issue keine Änderungen geben.

Reply:
restored, see 53cb0df838ef2eea537cf883342c672f06d9ba33